charts/janssen-all-in-one/templates/nginx-ingress.yaml

```diff
@@ -662,4 +662,3 @@ spec:
                 port:
                   number: 8080
 {{- end }}
-{{- end }}
```

Comment:
This is the closing `end` of main `if` clause. It should not be deleted. Instead remove line 615. See https://github.com/JanssenProject/jans/commit/4ad8fc5b14f7ffcfac152d2f75de8e136cb0fa0a for comparison.

Note, this file can be reverted.